website/README.md

```diff
@@ -0,0 +1,21 @@
+# Website
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+Website generation is done by
+[gh_pages_deploy.yaml](/.github/workflows/gh_pages_deploy.yaml). It runs
+`prebuild.py`, which prepares files for website generation, then builds the
+website using Jekyll, configured through `_config.yml`.
```

Comment:
Just hoping to document a bit the intended scope of the current effort, mostly to avoid confusion from external folks maybe looking at this part of the project...

```suggestion
Currently, the main website for Carbon is the GitHub project page as this
is still an experiment and in early stages of development.

However, we do maintain a documentation site where our project's documentation is rendered and more easily navigated. The documentation site generation is done by
[gh_pages_deploy.yaml](/.github/workflows/gh_pages_deploy.yaml). It runs
`prebuild.py`, which prepares files for website generation, then builds the
website using Jekyll, configured through `_config.yml`.
```

Happy to iterate on better phrasing though.

Reply:
I've added a paragraph along these lines. Note though, I really want to keep the "how does this work" section focused on that.

Reply:
Note, if there are more comments, maybe we could meet to run through details and get things resolved?